test/unit/gateways/mercury_test.rb

```diff
@@ -126,7 +126,7 @@ def test_transcript_scrubbing
 
   def successful_purchase_response
     <<~RESPONSE
-      <?xml version="1.0" encoding="utf-8"?><soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"><soap:Body><CreditTransactionResponse xmlns="http://www.mercurypay.com"><CreditTransactionResult><?xml version="1.0"?>
+      <?xml version="1.0" encoding="utf-8"?><soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"><soap:Body><CreditTransactionResponse xmlns="http://www.mercurypay.com"><CreditTransactionResult>
```

Comment:
more validation from https://github.com/ruby/rexml/pull/162

Reply:
This might break mercury gateway, but not sure if this is still used. http://www.mercurypay.com is no longer accessible. 

Running this in `master` branch is failing anyway. 
`ruby -Itest test/remote/gateways/remote_mercury_test.rb`